build.gradle.kts

```diff
@@ -60,6 +61,9 @@ dependencies {
     implementation("io.arrow-kt", "arrow-fx", arrowVersion)
     implementation("com.beust", "klaxon", "5.4")
     implementation("com.mojang", "brigadier", "1.0.17")
+    implementation("io.ktor", "ktor-client-core", ktorVersion)
```

Comment:
Instead of adding the full ktor library just for a call, I would use HttpUrlConnection like you do in java, ktor its pretty big and they break their API all the time

Reply:
and we can modify the download function instead to get a input stream